drivers/espi/host_subs_npcx.c

```diff
@@ -897,9 +897,9 @@ int npcx_host_periph_write_request(enum lpc_peripheral_opcode op,
 			return -ENOTSUP;
 		}
 		if (data) {
-			LOG_INF("%s: op 0x%x data %x", __func__, op, *data);
+			LOG_DBG("%s: op 0x%x data %x", __func__, op, *data);
 		} else {
-			LOG_INF("%s: op 0x%x only", __func__, op);
+			LOG_DBG("%s: op 0x%x only", __func__, op);
 		}
```

Comment:
Function prefix is already built into logging framework so you should drop this part from the macro invocations

Reply:
Done.